docs/content/doc/advanced/config-cheat-sheet.en-us.md

```diff
@@ -173,8 +173,9 @@ Values containing `#` or `;` must be quoted using `` ` `` or `"""`.
 - `LOG_SQL`: **true**: Log the executed SQL.
 - `DB_RETRIES`: **10**: How many ORM init / DB connect attempts allowed.
 - `DB_RETRY_BACKOFF`: **3s**: time.Duration to wait before trying another ORM init / DB connect attempt, if failure occured.
-- `MAX_IDLE_CONNS` **0**: Max idle database connections on connnection pool, default is 0
-- `CONN_MAX_LIFETIME` **3s**: Database connection max lifetime
+- `MAX_IDLE_CONNS` **2**: Max idle database connections on connnection pool, default is 2
+- `CONN_MAX_LIFETIME` **0 or 3s**: Database connection max lifetime - default is 0, meaning there is no limit (except on MySQL where it is 3s)
```

Comment:
The way this is expressed seems to suggest that if a transaction takes more than three seconds, it will be aborted.

Reply:
Heh it's just what was there before...

https://golang.org/pkg/database/sql/#DB.SetConnMaxLifetime
Says SetConnMaxLifetime sets the maximum amount of time a connection may be reused.

How about:

```suggestion
- `CONN_MAX_LIFETIME` **0 or 3s**: Maximum amount of time a DB connection may be reused - default is 0, meaning there is no limit (except on MySQL where it is 3s)
```

Reply:
```suggestion
- `CONN_MAX_LIFETIME` **0 or 3s**: sets the maximum amount of time a DB connection may be reused - default is 0, meaning there is no limit (except on MySQL where it is 3s)
```